images/build/Dockerfiles/Dockerfile

```diff
@@ -84,12 +83,6 @@ RUN set -ex \
     && yarnCacheFolder="/usr/local/share/yarn-cache" \
     && mkdir -p $yarnCacheFolder \
     && chmod 777 $yarnCacheFolder \
-    
-    # Install Golang SDKs
-    && . $buildDir/__golangVersions.sh \
-    && $imagesDir/installPlatform.sh golang $GOLANG_VERSION \
-    && ln -sfn $GOLANG_VERSION 1.16 \
-
```

Comment:
we dont need it here as we build this image from lts-version layer so we already have support for golang by now